trace-utils/src/msgpack_decoder/v04/decoder/mod.rs

```diff
@@ -47,49 +47,56 @@ use tinybytes::{Bytes, BytesString};
 /// };
 /// let encoded_data = to_vec_named(&vec![vec![span]]).unwrap();
 /// let encoded_data_as_tinybytes = tinybytes::Bytes::from(encoded_data);
-/// let decoded_traces = from_slice(encoded_data_as_tinybytes).expect("Decoding failed");
+/// let (decoded_traces, _) = from_slice(encoded_data_as_tinybytes).expect("Decoding failed");
```

Comment:
```suggestion
/// let (decoded_traces, _payload_size) = from_slice(encoded_data_as_tinybytes).expect("Decoding failed");
```
I think it's helpful to at least name the variable, even if we don't use it in the example.